### PR TITLE
allow android video upload by adding an extra menu to specify image/video CORE-8489

### PR DESCRIPTION
**Pull request by mmaxim**
`react-native-image-picker` doesn't allow `mixed` mode for Android like it does for iOS, so selecting video vs. photo is a little tricker. This patch adds an extra screen after clicking the photo attachment to allow the user to specify image vs. video, and puts the picker into the mode they select. 

@keybase/react-hackers

### CODE REVIEW

**Comment by chrisnojima** [comment]
Why not just have android have 2 options in the original menu. Upload picture / upload video

File: shared/chat/conversation/input-area/filepicker-popup/index.native.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+  onDone = () => {
+    this.props.onSelect(this.state.selected)
+    this.props.onHidden()
+  }
+
+  render() {
+    const items = [{label: 'Send an image', value: 'photo'}, {label: 'Send a video', value: 'video'}]
+    return (
+      <FloatingPicker
```
Comment (buoyad) [inline shared/chat/conversation/input-area/filepicker-popup/index.native.js]: This should use `FloatingMenu`

File: shared/chat/conversation/input-area/normal/platform-input.native.js [hunk 2]
```diff
@@ -45,7 +49,17 @@ class PlatformInput extends Component<PlatformInputProps & OverlayParentProps, S
...
+      this._showNativeImagePicker('mixed')
+    } else {
+      // On Android "mixed" mode doesn't work with our library, so we have to ask the user
+      // if they want photo or video
+      this._toggleShowingMenu('filepickerpopup')
+    }
+  }
+
+  _showNativeImagePicker = (mediaType: string) => {
+    showImagePicker({mediaType}, response => {
```
Comment (buoyad) [inline shared/chat/conversation/input-area/normal/platform-input.native.js]: Can you change the `Title`s in this options object in the video case?
image

**Comment by mmaxim** [comment]
@buoyad PTAL!

File: shared/chat/conversation/input-area/filepicker-popup/index.native.js [hunk 3]
```diff
@@ -1,11 +1,12 @@
 // @flow
 import * as React from 'react'
-import {Box2, Text, FloatingPicker} from '../../../../common-adapters/mobile.native'
+import {FloatingMenu} from '../../../../common-adapters'
+import {Box2, Text} from '../../../../common-adapters/mobile.native'
```
Comment (buoyad) [inline shared/chat/conversation/input-area/filepicker-popup/index.native.js]: These can be in the same import as above

**Pull request merged**